hikari/api/rest.py

```diff
@@ -882,7 +882,7 @@ async def edit_message(
             a `hikari.channels.TextChannel` or the ID of an existing
             channel.
         message : hikari.snowflakes.SnowflakeishOr[hikari.messages.Message]
-            The message to edit. This may be a `hikari.messages.Message`
+            The message to edit. This may be a message object
```

Comment:
define "message object"? Does that include partial messages? (The answer is yes).